test/common/tests.go

```diff
@@ -29,6 +29,6 @@ var (
 		{"C01 - Verify Alerts are not firing apart from DeadMansSwitch", TestIntegreatlyAlertsFiring},
 		{"B05 - Verify Codeready CRUDL permissions", TestCodereadyCrudlPermisssions},
 		{"A09 - Verify Subscription Install Plan Strategy", TestSubscriptionInstallPlanType},
-		{"B06 - Verify users with no email get default email", TestDefaultUserEmail},
+		{"Verify users with no email get default email", TestDefaultUserEmail},
```

Comment:
Would be good to follow Davides steps and see if this is replacing a current manual test, the links are at the top of this file